src/creature.cpp

```diff
@@ -179,11 +179,24 @@ bool Creature::sees( const Creature &critter ) const
                     abs( posz() - critter.posz() ) <= 1 ) ) ) {
         return false;
     }
+    if( critter.is_player() && g->u.move_mode == "crouch" ) {
+        float coverage = g->m.obstacle_coverage( pos(), critter.pos() );
```

Comment:
`obstacle_coverage` returns `int`. Why the assignment to `float`?

Reply:
Thanks, didn't notice it.